packages/salesforcedx-apex-debugger/tsconfig.json

```diff
@@ -2,22 +2,16 @@
   "compilerOptions": {
     "module": "commonjs",
     "target": "es6",
-    "lib": [
-      "dom",
-      "es2016"
-    ],
+    "lib": ["dom", "es2016"],
     "sourceMap": true,
     "declaration": true,
     "moduleResolution": "node",
     "noImplicitAny": true,
     "rootDir": ".",
     "outDir": "out",
     "preserveConstEnums": true,
-    "strict": true
+    "strict": true,
+    "strictPropertyInitialization": false
```

Comment:
I opened work item W-5676805 to refactor our code to conform to the "strictPropertyInitialization" rule.

Reply:
@allileong - I'm going to work on W-5676805 in GUS.  Do you need me to open up an issue for that on github, or is it sufficient for me to reference W-5676805 in my PR without a matching Github Issue?

Reply:
@dehru - There's no need to create a separate github issue. Referencing W-5676805 in your PR will be sufficient. Thanks for working on this item!